config.py

```diff
@@ -0,0 +1,30 @@
+import os
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+
+class Config:
+    """Base configuration variables."""
+    TESTING = False
+    TRELLO_KEY = os.environ.get('TRELLO_KEY')
+    TRELLO_TOKEN = os.environ.get('TRELLO_TOKEN')
+    TRELLO_BOARD = os.environ.get('TRELLO_BOARD')
+
+
+class TestingConfig(Config):
+    """Test configuration variables."""
+    TESTING = True
+    #TRELLO_BOARD = os.environ.get('TRELLO_TEST_BOARD')
```

Comment:
For the integration test, if we are sticking with this system we probably want to override the key/token so that if we fail to mock a request, we don't accidentally find ourselves updating the actual trello board

Reply:
will change to try the alternative approach of different config files, but would have a test config as the default ordinarily and the prod one for the deployment